runtime/sam/op/ztests/distinct.yaml

```
@@ -0,0 +1,29 @@
+script: |
+  super -z -c 'from "1.jsup" | distinct x'
+  echo ===
+  super -z -c 'from "2.jsup" | distinct abs(this)'
+
+inputs:
+  - name: 1.jsup
+    data: |
+      {x:1}
+      {x:2}
+      {x:2}
+      {y:3}
```

Nit: Add a named type.

```suggestion
      {y:3}
      {y:3}(=named)
```